### DISCUSSION TITLE
Read the Ingress metric from ingress pod directly

### DISCUSSION THREAD

**Discussion by arjaiswa** [nginx/kubernetes-ingress#1871] [General]
Currently Ingress metric can be fetch from 911/metric endpoint.
Do we have any way to read metric from ingress Pod directly if my other pod is in the same cluster .
Or Can we scrape the metric directly to other alerting resources ?

**Comment by pleshakov** [comment]
the stub_status metrics can be accessed from another pod in a cluster -- https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/status-page/#accessing-stub-status 

additional IC-related metrics like `controller_nginx_reloads_total` are only available via  the Prometheus :9113/metrics endpoint -- https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/prometheus/ . They could be scrapped by other pods in the same cluster.

**Comment by arjaiswa** [comment]
Thank you for the information . 
If in kubernetes cluster I have 1 controller and multiple ingress rule in different namespace . In metric can we get this information 
How many active connection ingress rule wise ?
example : 
Ingress controller
     Ingressrule1 : 2 active
     Ingressrule2 : 3 active

**Comment by pdabelf5** [comment]
What you are asking for is not currently possible.  Please create a feature request issue if this is still a use case.
